src/directives/markers.js

```diff
@@ -269,6 +269,9 @@ angular.module('ui-leaflet').directive('markers',
                             return;
                         _create(newMarkers, oldMarkers);
                     });
+                    scope.$on('$destroy', function () {
+                        _destroy(leafletScope.markers, {}, leafletMarkers, map, layers);
```

Comment:
Thanks for merging it. I still have a question: in this line, we only delete the markers that are not anymore in the scope (if I understand correctly what `_seeWhatWeAlreadyHave` does). But if we destroy the directives, shouldn't we delete all the leaflet markers?